### PR TITLE
Creatives New and Edit

### PR DESCRIPTION
**Pull request by cmckee-dev**
## Creatives New and Edit Redesign

**Index:**  

creatives-index

**New:**  

creatives-new  

---

**Edit with Standard tab:**  

creatives-edit-standard

---

**Edit with Sponsor tab:**  

creatives-edit-sponsor

### CODE REVIEW

**Comment by andrewmcodes** [comment]
Is this ready for review @cmckee-dev?

File: app/views_redesigned/ad_templates/@responsive_footer/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<% cache_if @campaign.persisted? && @creative.persisted?, [@campaign, @creative] do %>
+  <div id="cf">
+    <a data-href="campaign_url" class="cf-img-wrapper" target="_blank" rel="nofollow noopener">
```
Comment (andrewmcodes) [inline app/views_redesigned/ad_templates/@responsive_footer/show.html.erb]: This is a change being ported over to the ad_templates in our current views folder but can you do us a favor and change `nofollow` to `sponsored` in each of the files under `app/views_redesigned/ad_templates/`?

File: app/views_redesigned/shared/actions/_add_images.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<%= link_to new_image_path(imageable.to_gid_param), class: "btn btn-sm btn-soft-white transition-3d-hover" do %>
```
Comment (andrewmcodes) [inline app/views_redesigned/shared/actions/_add_images.html.erb]:
```suggestion
<%= link_to new_image_path(imageable.to_gid_param), class: "btn btn-sm btn-subtle-primary" do %>
```

File: app/views_redesigned/ad_templates/@responsive_footer/show.html.erb (hunk 1 above)
Comment (cmckee-dev) [reply app/views_redesigned/ad_templates/@responsive_footer/show.html.erb]: Sure! I saw a related PR, just want to double check this wasnt finished up. And I'm not seeing a `ad_templates/` directory under master. Do you want me to copy that over?

Comment (cmckee-dev) [reply app/views_redesigned/ad_templates/@responsive_footer/show.html.erb]: :heavy_check_mark:

File: app/views_redesigned/creatives/forms/_standard_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+      <%#             required> %>
+      <%#             <option value="">--- select an image ---<option> %>
+      <%#             <% current_user.wide_images.each do |image| %1> %>
+      <%#               <%= tag.option Image.new(image).display_name, value: image.blob_id, %>
+      <%#                 data: { image_url: polymorphic_url(image) }, %>
+      <%#                 selected: creative.wide_image == image %1> %>
+      <%#             <% end %1> %>
+      <%#     </select> %>
+      <%#   </div> %>
+      <%# </div> %>
```
Comment (andrewmcodes) [inline app/views_redesigned/creatives/forms/_standard_form.html.erb]: Can we delete this commented out code?

Comment (cmckee-dev) [reply app/views_redesigned/creatives/forms/_standard_form.html.erb]: :heavy_check_mark:

**Pull request merged**